src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -206,6 +206,11 @@ public Player generatePlayer(){
         return new Player(generatedId, username, token, team);
     }
 
+    // TODO: Only for testing, feel free to reimplement with corresponding story.
+    public void addPlayer(Player player) {
+        playerMap.put(player.getToken(), player);
```

Comment:
This should use generatePlayer and leave generatePlayer private, but as we want to rewrite it using a Playerfactor it is fine for now.